contracts/libs/RollupUtils.sol

```diff
@@ -692,12 +681,12 @@ library RollupUtils {
     function BurnConsentSignBytes(
         uint256 txType,
         uint256 fromIndex,
-        uint256 amount,
-        uint256 nonce
+        uint256 nonce,
+        uint256 amount
```

Comment:
@vaibhavchellani want to confirm if we want this

Reply:
Should be okay, ill update the client. But thanks for pointing out.